app/Services/Mailer/SendGrid.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace App\Services\Mailer;
+
+use SendGrid as API;
+use SendGrid\Mail\Mail;
+use App\Services\Security\TokenDependent;
+use App\Models\Setting;
+use App\Models\EmailLog;
+
+class SendGrid extends Mailer
+{
+    use TokenDependent;
+
+    function __construct()
+    {
+        $this->email = new Mail();
+        $this->response = null;
+
+        $this->email->setFrom(Setting::ofGroup(Setting::GROUP_MAIL)->withKey(Setting::KEY_FROM_EMAIL)->first()->value,
```

Comment:
Please create static helper for the code below on the Setting model that accepts group & key and returns the value.

`Setting::ofGroup(Setting::GROUP_MAIL)->withKey(Setting::KEY_FROM_EMAIL)->first()->value`